demo/fit_a_line/train.py

```diff
@@ -2,38 +2,57 @@
 import paddle.v2.dataset as dataset
 import os
 import gzip
+import sys
 
-#PaddleCloud cached the dataset on /pfs/${DATACENTER}/public/dataset/...
-dc = os.getenv("PADDLE_CLOUD_CURRENT_DATACENTER")
-dataset.common.DATA_HOME = "/pfs/%s/public/dataset" % dc
+# NOTE: You should change USERNAME to your account.
+# TODO(Yancey1989): fetch username from environment variable.
+USERNAME = "yanxu05@baidu.com"
 
-trainer_id = int(os.getenv("PADDLE_INIT_TRAINER_ID"))
-trainer_count = int(os.getenv("PADDLE_INIT_NUM_GRADIENT_SERVERS"))
+DC = os.getenv("PADDLE_CLOUD_CURRENT_DATACENTER")
 
-# TODO(helin): remove this once paddle.v2.reader.creator.recordio is
-# fixed.
-def recordio(paths, buf_size=100):
+#PaddleCloud cached the dataset on /pfs/${DATACENTER}/home/${USERNAME}/...
+dataset.common.DATA_HOME = "/pfs/%s/home/%s" % (DC, USERNAME)
+TRAIN_FILES_PATH = os.path.join(dataset.common.DATA_HOME, "uci_housing")
+
+def prepare_dataset():
+    dataset.common.convert(TRAIN_FILES_PATH,
+                           dataset.uci_housing.train(), 100, "train")
+
+TRAINER_ID = int(os.getenv("PADDLE_INIT_TRAINER_ID"))
+TRAINER_COUNT = int(os.getenv("PADDLE_INIT_NUM_GRADIENT_SERVERS"))
```

Comment:
`TRAINER_COUNT` can be confused with the trainer argument, but it's OK, refine this if you have time.

Reply:
Done, maybe `TRAINER_INSTANCES` is more suitable.